_sass/core/_defaults.scss

```diff
@@ -3,7 +3,7 @@
 // These variables can be modified in _variables.scss
 
 // Typography
-$em-base:             16px; // default is already defined in bourbon
+$em-base:             10px; // default is already defined in bourbon
```

Comment:
@emilyville Maybe why everything looks bigger? I wonder why they made this change without changing all the other sizes, seems pretty drastic